.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk6
+  - openjdk7
```

Comment:
it was something with long hostnames on their new platform. Doesn't look like it affects you.

more here : https://github.com/travis-ci/travis-ci/issues/5669

Probably not a real concern though. It'll show up in the build logs